package.json

```diff
@@ -19,7 +19,7 @@
     "build": "next build",
     "prod": "NODE_ENV=production node server.js",
     "pretest": "eslint ./src",
-    "test": "jest",
+    "test": "jest --silent",
```

Comment:
what does this flag do?

Reply:
it suppresses `console` messages in the tests so it's easier to browse through in a terminal with limited buffer. there is also a `test:loud` that makes those messages appear. this is similar in behavior to how the `api` works now.